frontend/src/app/shared/components/sidenav/sidenav.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../../../sass/color';
+
+sidenav-layout {
+    box-sizing: border-box;
+    display: block;
+    height: 100%;
+}
+sidenav-layout.sidenav-shown {
```

Comment:
Yeah you can do that, you'll just need to do `&.sidenav-shown` instead of just `.sidenav-shown`.